src/javascripts/ng-admin/Crud/button/maCreateButton.js

```diff
@@ -15,7 +15,13 @@ define(function () {
                 scope.label = scope.label || 'Create';
 
                 scope.gotoCreate = function () {
-                    $state.go($state.get('create'), { 'entity': scope.entity().name() });
+                    $state.go($state.get('create'), {
```

Comment:
Why not merging `$state.params?` like `angular.extend($state.params, {/* ... */});` ?